Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -31,6 +32,12 @@ class vtkMatrix4x4;
 class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
 {
   public:
+  /// \brief Instantiate a new Slice node without any orientation presets.
+  ///
+  /// \note To instantiate a vtkMRMLSliceNode with preconfigured
+  /// orientation preset matrices (the default presets are: Axial,
+  /// Sagittal and Coronal in default), it is necessary to use
+  /// vtkMRMLScene::CreateNodeByClass(const char*)
```

Comment:
@lassoan There is one caveat of relying on the default node infrastructure, slide node created directly (without the help of `vtkMRMLScene::CreateNodeByClass`) will not have any default orientation presets.

It looks like this will not be a problem, in Slice core, the instance directly created do not make use of the orientation:

```
$ ack --py "vtkMRMLSliceNode\("
Modules/Scripted/PerformanceTests/PerformanceTests.py
304:    self.sliceNode = slicer.vtkMRMLSliceNode()

$ fgrep -rnsI "vtkMRMLSliceNode>::N" .
./Modules/Loadable/Annotations/MRML/vtkMRMLAnnotationNode.cxx:737:  this->m_RedSliceNode = vtkSmartPointer<vtkMRMLSliceNode>::New();
./Modules/Loadable/Annotations/MRML/vtkMRMLAnnotationNode.cxx:738:  this->m_YellowSliceNode = vtkSmartPointer<vtkMRMLSliceNode>::New();
./Modules/Loadable/Annotations/MRML/vtkMRMLAnnotationNode.cxx:739:  this->m_GreenSliceNode = vtkSmartPointer<vtkMRMLSliceNode>::New();

$ fgrep -rnsI "vtkMRMLSliceNode::N" .
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLSliceViewDisplayableManagerFactoryTest.cxx:59:  vtkMRMLSliceNode * viewNode = vtkMRMLSliceNode::New();

$ fgrep -rnsI "vtkNew<vtkMRMLSliceNode>" .
./Modules/Loadable/Annotations/Testing/Cxx/vtkMRMLAnnotationDisplayableManagerTest2.cxx:73:  vtkNew<vtkMRMLSliceNode> viewNode;
./Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx:306:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Widgets/Testing/qMRMLDisplayNodeViewComboBoxTest.cxx:67:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Widgets/Testing/qMRMLSliceControllerWidgetTest.cxx:87:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:39:  vtkNew<vtkMRMLSliceNode> node1;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:98:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:137:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:173:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:236:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:254:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:300:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSliceNodeTest1.cxx:326:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Core/Testing/vtkMRMLSceneIDTest.cxx:157:  vtkNew<vtkMRMLSliceNode> redSlice;
./Libs/MRML/Core/Testing/vtkMRMLSceneIDTest.cxx:168:  vtkNew<vtkMRMLSliceNode> greenSlice;
./Libs/MRML/Core/Testing/vtkMRMLSceneIDTest.cxx:180:  vtkNew<vtkMRMLSliceNode> greenSlice2;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLThreeDReformatDisplayableManagerTest1.cxx:927:  vtkNew<vtkMRMLSliceNode> sliceNodeRed;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLThreeDReformatDisplayableManagerTest1.cxx:935:  vtkNew<vtkMRMLSliceNode> sliceNodeYellow;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLThreeDReformatDisplayableManagerTest1.cxx:943:  vtkNew<vtkMRMLSliceNode> sliceNodeGreen;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLThreeDReformatDisplayableManagerTest1.cxx:953:  vtkNew<vtkMRMLSliceNode> sliceNodeDeleted;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLThreeDReformatDisplayableManagerTest1.cxx:962:  vtkNew<vtkMRMLSliceNode> sliceNodeImported;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLThreeDReformatDisplayableManagerTest1.cxx:973:  vtkNew<vtkMRMLSliceNode> sliceNodeRestored;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLDisplayableManagerFactoriesTest1.cxx:162:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/DisplayableManager/Testing/Cxx/vtkMRMLModelSliceDisplayableManagerTest.cxx:87:  vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Logic/Testing/Cxx/vtkMRMLLayoutLogicTest2.cxx:27:  vtkNew<vtkMRMLSliceNode> redSliceNode;
./Libs/MRML/Logic/Testing/Cxx/vtkMRMLLayoutLogicTest2.cxx:31:  vtkNew<vtkMRMLSliceNode> yellowSliceNode;
./Libs/MRML/Logic/Testing/Cxx/vtkMRMLLayoutLogicTest2.cxx:35:  vtkNew<vtkMRMLSliceNode> greenSliceNode;
./Libs/MRML/Logic/Testing/Cxx/vtkMRMLApplicationLogicTest1.cxx:111:    vtkNew<vtkMRMLSliceNode> sliceNode;
./Libs/MRML/Logic/Testing/Cxx/vtkMRMLSliceLogicTest1.cxx:39:  vtkNew<vtkMRMLSliceNode> SliceNode;
``` 

and I could not find any in Slice extensions. (Using these pattern `.vtkMRMLSliceNode(`, `vtkMRMLSliceNode::New`,   `vtkMRMLSliceNode>::New` or `vtkNew<vtkMRMLSliceNode>`